.github/workflows/ci.yml

```diff
@@ -35,13 +35,10 @@ jobs:
     name: Build And Run linter
     runs-on: ubuntu-latest
     steps:
-      -
-        uses: actions/checkout@v4
-      -
-        name: Set up Docker Buildx
+      - uses: actions/checkout@v4
```

Comment:
If we change the style here, I would favor changing it for the whole file. So also in the section endToEndTests

Reply:
Huh, my Clion changed it actually to what I have there now. There was an update to clang tidy, do you have that as well?
Can you double check, I will revert it, if spurious on my part

Reply:
Hm... It did the same for me. So we can leave this as is.